### DISCUSSION TITLE
[request] OpenGL Spectrum Analyzer modification

### DISCUSSION THREAD

**Discussion by atomGit** [audacious-media-player/audacious#76] [Q&A]
wondering if anyone would be willing to mod the GL spectrum analyzer to eliminate the rotation and make it consume all of the window space?

Christophe Budé, John Lindgren, and Carlo Bramini seem to be the developers but i don't know how to contact them and issues are not enabled on the [audacious-plugins repo](https://github.com/audacious-media-player/audacious-plugins)

**Comment by dualword** [comment]
[link](https://github.com/dualword/audacious-plugins/tree/audacious-plugins-mod-4.3.1/src/qtglspectrum1)

**Comment by radioactiveman** [comment]
This could be made configurable with a checkbox in the plugin settings.

@dualword: Would you be interested in implementing this and submitting a pull request?

**Comment by radioactiveman** [comment]
Filed now as feature request #1576.
